superbench/tools/system_info.py

```diff
@@ -437,4 +442,11 @@ def get_all(self):
         sum_dict['Network'] = self.get_network()
         sum_dict['PCIe'] = self.get_pcie()
         sum_dict['Accelerator'] = self.get_gpu()
+        with open('system.json', 'w', encoding='utf-8') as f:
+            json.dump(sum_dict, f, indent=4)
```

Comment:
remove

Reply:
Who will help dump the json?

Reply:
you can `sb node info > output.json`
or `sb node info -o yaml > output.yaml`, `sb node info -o tsv > output.tsv`, etc.

Reply:
@yukirora would you please remove it?